client/src/screens/ExportElectionBallotPackageScreen.tsx

```diff
@@ -190,6 +137,7 @@ const ExportElectionBallotPackageScreen = () => {
             election={election}
             precinctId={precinctId}
             onRendered={onRendered}
+            isLiveMode={!testMode}
```

Comment:
nit: toggle this var to be `isLiveMode` instead of `testMode`?

Reply:
Done.